docs/interpreter/springxd.md

```diff
@@ -0,0 +1,150 @@
+* A Paragraph Run will `Create`, automatically `Deploy` and `Start` the jobs defined in the Paragraph.
+* If a job deployment fail then all jobs in the paragraph are destroyed.
+* Zeppelin returns after the jobs have started (e.g. Zeppelin goes into `Finished` state).
+* When job have successfully been deployed the result contains a button that lists the just deployed jobs and status `DEPLOYED`
+* To destroy all jobs in the paragraph, press the `Annular button` in the result section. Button state should switch from `DEPLOYED` to `DESTROYED`.
+* Jobs can refer other streams or jobs in any paragraph and even different notebooks.
+
+###### Job Examples:
+Example module which loads [CSV files into a JDBC table](http://docs.spring.io/spring-xd/docs/current/reference/html/#file-jdbc) using a single batch job.
+
+```
+%xd.job
+myJob = filejdbc --resources=file:///mycsvdir/*.csv --names=forename,surname,address --url=jdbc:mysql://localhost:3306/myTable --username=jdbcUser --password=jdbcPassword --tableName=people --initializeDatabase=true
+``` 
+The job should be defined with the resources parameter defining the files which should be loaded. It also requires a names parameter (for the CSV field names) and these should match the database column names into which the data should be stored. You can either pre-create the database table or the module will create it for you if you use --initializeDatabase=true when the job is created. The table initialization is configured in a similar way to the JDBC sink and uses the same parameters. The default table name is the job name and can be customized by setting the tableName parameter. As an example, if you run the command
+
+Another example illustrates how a Spark Application can be deployed and launched from Spring XD as a batch job. SparkTasklet submits the Spark application into Spark cluster manager using org.apache.spark.deploy.SparkSubmit:
+
+```
+%xd.job
+SparkPiExample  = sparkapp --appJar=<the location of spark-examples-1.2.1 jar> --name=MyApp --master=<spark master url or local> --mainClass=org.apache.spark.examples.SparkPi
+```
+
+#### Apply Zeppelin Dynamic Forms
+
+You can leverage [Zeppelin Dynamic Form](../manual/dynamicform.html) inside your stream and job definitions. You can use both the `text input` and `select form` parameterization features. For example to parameterize the search parameter in the twitter search stream :
+
+```
+%xd.stream
+
+tweets = twittersearch --query=${MySearchParameter=DefaultSearchKeyword} --outputType=application/json | hdfs
+``` 
+
+
+### Auto-completion
+The SpringXD Interpreter provides a comprehensive auto-completion for `Job` and `Stream` DSL definitions. On `(Ctrl+.)` it list the most relevant suggestions in a pop-up window. Implementation leverages [SpringXD's Completion API](http://docs.spring.io/spring-xd/docs/current/reference/html/#completions) and provides the same expressiveness similar with the SpringXD Shell.
```

Comment:
I think "On `(Ctrl+.)` it list the most" -> "On `(Ctrl+.)` it lists the most" may be correct : )